codec-http2/src/main/java/io/netty/handler/codec/http2/Http2Headers.java

```diff
@@ -68,12 +68,25 @@ enum PseudoHeaderName {
 
         private final AsciiString value;
         private final boolean requestOnly;
-        private static final CharSequenceMap<PseudoHeaderName> PSEUDO_HEADERS = new CharSequenceMap<PseudoHeaderName>();
+
+        /**
+         * This has been implemented in order to allow the JIT
+         * to produce a binary-search like sequence of conditional
+         */
+        private static final int METHOD_HASH_CODE = -240705714;
```

Comment:
These requires to add a version based on big Indian as well :/

Sadly ASCII hashcode uses Unsafe under the hood, which depends on endianess :(

Reply:
yeah, long outstanding bug with the endianess